test/egress/testdata/proxy.yaml

```diff
@@ -165,7 +165,7 @@ spec:
 apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
-  name: egress-test-http-get
+  name: egress-test-not-www-get
```

Comment:
Ugh are you serious? This test has been reconfiguring an existing deployment the whole time?! Wow. Thanks for finding and fixing it!

Reply:
I was trying to figure out why the tests kept failing and it turned out to be this. 😩